src/booleans.js

```diff
@@ -1,62 +1,66 @@
 function negate(a) {
-  // your code here
-};
+  return !a;
+}
 
 function both(a, b) {
-  // your code here
-};
+  return a && b;
+}
 
 function either(a, b) {
-  // your code here
-};
+  return a || b;
+}
 
 function none(a, b) {
-  // your code here
-};
+  return !a && !b;
+}
 
 function one(a, b) {
-  // your code here
-};
+  return (a && !b) || (!a && b);
+}
 
 function truthiness(a) {
-  // your code here
-};
+  if (a) { 
+    return true; } 
+    else {
+    return false; }  
+}
 
 function isEqual(a, b) {
-  // your code here
-};
+  return a === b;
+}
 
 function isGreaterThan(a, b) {
-  // your code here
-};
+  return a > b;
+}
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
-};
+  return a <= b;
+}
 
 function isOdd(a) {
-  // your code here
-};
+  return a % 2 !== 0 ? true : false;
+}
 
 function isEven(a) {
-  // your code here
-};
+  return a % 2 === 0 ? true : false;
+}
 
 function isSquare(a) {
-  // your code here
-};
+  return Math.sqrt(a) % 1 === 0;
+}
 
 function startsWith(char, string) {
-  // your code here
-};
+  return string[0] === char ? true : false;
+}
```

Comment:
Instead of using the tertiary operator `?`, and returning `true` or `false` depending on what the condition evaluates to, just return the condition. e.g.
```
return a % 2 !== 0
```